oss-cloud-frontend/src/App.vue

```diff
@@ -0,0 +1,36 @@
+<template>
```

Comment:
Let's introduce router here, and redirect to HelloWorld component from it.

Reply:
Let's do that in a separate PR, this one is already covering too many things.

Reply:
We would like to introduce routing in some later pull request, when we start developing frontend app. HelloWorld page is just a placeholder for now.